Update index.html

abckf654 opened the pull request:
Change celsius range to 36.1 ~ 36.9 to align with realistic summer time temperature.

abckf654 committed:
Update index.html

Change celsius range to 36.1 ~ 36.9 to align with realistic summer time temperature.

TsumaHibiki commented:
Changing the body temp manually due to seasons is inconvenient.
A new update for adjusting the range automatically based on seasons is required.

123standup commented:
> Changing the body temp manually due to seasons is inconvenient. A new update for adjusting the range automatically based on seasons is required.
Great recommendation. It can be left in future improvement queue, or you can pull a new request to complete.

TsumaHibiki committed:
Update range and add seasonal random body temp

add function getMonthlyAvgTemp including the monthly temp of Taipei
change the algo based on this monthly avg such that the standard body temp is calculated by the ratio of the current monthly avg and the random body temp is adjusted between 0 - 0.5
